app/kubernetes-ingress-controller/faq/migrate-ingress-to-gateway.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrating from Ingress to Gateway API
+
+description: |
+  Which custom Ingress annotations are replaced with Gateway API features?
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+Kong contributed to the kubernetes-sigs project [`ingress2gateway`](https://github.com/kubernetes-sigs/ingress2gateway) by creating a Kong provider able to convert ingress resources into Gateway resources. The `ingress2Gateway` tool provides a unique `print` command that gets ingress resources and displays the Gateway API equivalent. The output of such an operation can be either directly applied to the cluster, or generate a new set of yaml files containing the converted configuration.
```

Comment:
```suggestion
Kong contributed to the kubernetes-sigs project [`ingress2gateway`](https://github.com/kubernetes-sigs/ingress2gateway) by creating a Kong provider able to convert ingress resources into Gateway resources. The `ingress2Gateway` tool provides a `print` command that gets ingress resources and displays the Gateway API equivalent. The output of this operation can be either directly applied to the cluster, or it can generate a new set of YAML files containing the converted configuration.
```